cli/common/socket.go

```diff
@@ -20,7 +20,7 @@ const (
 )
```

Comment:
Would be great to move this changes into separate commit. With good description and brilliant commit message.

Reply:
See #425